src/test/java/com/cloudbees/jenkins/plugins/amazonecs/ECSCloudTest.java

```diff
@@ -184,6 +184,7 @@ private ECSTaskTemplate getTaskTemplate(String templateName, String label) {
                 templateName,
                 label,
                 "",
+                "",
```

Comment:
I noticed there are null checks for this field in `ECSTaskTemplate.java`. Does it make sense to test here with `null` instead of an empty string?

Reply:
Since the new `agentContainerName` is tied so closely to `taskDefinitionOverride`, I tried to duplicate the latter's values in tests when the reason for the value was non-obvious. So, if `taskDefinitionOverride` was `null` I also made `agentContainerName` null and likewise when it was an empty string.